### PR TITLE
bodyarea-footer

### PR DESCRIPTION
**Pull request by hieunguyen64**
Reconstruct Project file.
Added body area and footer section.
Making carousel with slick slider.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,46 +113,224 @@
...
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <script src="https://code.jquery.com/jquery-2.2.0.min.js" type="text/javascript"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
+    <script type="text/javascript" src="js/main.js"></script>
```
Comment (thienhophu) [inline index.html]: Move these to top of the document

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,6 @@
 
 requirement/.~lock.ProjectEstimate.excel#
+
+js/slick-1.8.1/
+
+slick/
```
Comment (thienhophu) [inline .gitignore]: remove unused lines

File: css/style.css [hunk 3]
```diff
@@ -30,7 +41,172 @@
...
+
+.slick-dots {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    padding: 10px 0;
+    list-style-type: none;
+    position: absolute;
+    bottom: 0;
+    right: 50%;
```
Comment (thienhophu) [inline css/style.css]: Move this on top of slick-dots. It should be talking from outside to inside

File: css/style.css [hunk 4]
```diff
@@ -30,7 +41,172 @@
...
+}
+
+.slick-dots li.slick-active button {
+    transform: scale(2.5);
+    box-shadow: 0px 3px 5px 1px rgba(0, 0, 0, 0.205);
+}
+
+
+/*Slider two*/
+
```
Comment (thienhophu) [inline css/style.css:102]: remove unused line

File: css/style.css [hunk 5]
```diff
@@ -30,7 +41,172 @@
...
+}
+
+.site-slider-two:hover .slider-btn .prev,
+.site-slider-two:hover .slider-btn .next {
+    opacity: 1;
+}
+
+.position-mid {
+    position: absolute;
+    top: 30%;
```
Comment (thienhophu) [inline css/style.css]: if this is in the middle of something. Then this logic is wrong and will be rendered incorrect on different size screens.

File: index.html [hunk 6]
```diff
@@ -103,46 +113,224 @@
...
-                    </ol>
-                    <div class="carousel-inner">
-                        <div class="carousel-item active">
-                            <img src="image/Carousel/carousel4.jpg" style="width: 900px; height: 400px;" class="d-block ">
-                            <div class="carousel-caption">
-                                <h5>Modern Style</h5>
-                            </div>
+                <div class="site-slider">
+                    <div class="slider-one">
+                        <div>
```
Comment (thienhophu) [inline index.html]: remove unused div

**Review by thienhophu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (hieunguyen64) [reply index.html]: done

File: .gitignore (hunk 2 above)
Comment (hieunguyen64) [reply .gitignore]: done

File: css/style.css (hunk 3 above)
Comment (hieunguyen64) [reply css/style.css]: done

File: css/style.css (hunk 4 above)
Comment (hieunguyen64) [reply css/style.css:102]: done

File: css/style.css (hunk 5 above)
Comment (hieunguyen64) [reply css/style.css]: It's the unused code and has been deleted.

File: index.html (hunk 6 above)
Comment (hieunguyen64) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (hieunguyen64) [reply index.html]: When I moved this to the top, the slick didn't work.

**Pull request merged**